src/styles/styles/_active.scss

```diff
@@ -0,0 +1,24 @@
+.is-active {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+  font-size: 12px;
+  white-space: nowrap;
+  transition: color 0.2s;
```

Comment:
It would be great to create a variable for transition duration so you know for sure that it's the same for the whole page